pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -148,6 +113,12 @@ func checkTrayBinaryVersion() error {
 }
 
 func fixTrayBinaryVersion() error {
+	// Kill older running tray
+	if err := checkTrayRunning(); err == nil {
+		if err := stopTray(); err != nil {
+			logging.Debugf("Failed to kill running tray: %v", err)
+		}
+	}
```

Comment:
This code used to be in `fixTrayBinaryExists`, not clear why it moved.

Reply:
Yes, but it should have been here all along, we can only expect a running tray when there is one available, and only when it is available and the versions don't match we need to update the version in which case we'll have to stop the running tray, in normal case when `fixTrayBinaryExists` is called, it'll most probably not have a running tray, as that binary doesn't exist yet

Reply:
can you move this to its own commit? The change to use `Cache` should be purely code moving around, with no behaviour change, the explanation you just gave deserves to be in a git commit log :)